rocky/tools/management/commands/generate_report.py

```diff
@@ -0,0 +1,101 @@
+from pathlib import Path
+
+import sys
+from datetime import datetime, timezone
+from logging import getLogger
+from typing import Optional, List, Dict, Any
+
+from django.contrib.auth import get_user_model
+from django.core.management import BaseCommand
+from octopoes.connector.octopoes import OctopoesAPIConnector
+from octopoes.models import DEFAULT_SCAN_LEVEL_FILTER, DEFAULT_SCAN_PROFILE_TYPE_FILTER
+from octopoes.models.ooi.findings import Finding
+
+from django.conf import settings
+from rocky.keiko import keiko_client, ReportsService
+from rocky.views.finding_list import generate_findings_metadata
+from rocky.views.mixins import OOIList
+from tools.models import Organization
+from tools.ooi_helpers import RiskLevelSeverity
+
+User = get_user_model()
+logger = getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = (
+        "Generates a Finding report for all findings in an organization. "
+        "Requires either the id (-i) or the unique code (-c) of an organization."
+    )
+
+    def add_arguments(self, parser):
+        parser.add_argument("--id", "-i", type=int, help="The primary key of the organization.")
+        parser.add_argument("--code", "-c", type=str, help="The unique organization code.")
+        parser.add_argument(
```

Comment:
Usually, in most CLI apps this option is consistently called `--output/-o`. Also, I think it's better to mention (in the help) that it writes to `stdout` by default, the user itself can decide what to do with it.